pcsx2/ImGui/ImGuiOverlays.cpp

```diff
@@ -666,8 +665,9 @@ void ImGuiManager::DrawInputRecordingOverlay()
 
 void ImGuiManager::RenderOverlays()
 {
-	DrawPerformanceOverlay();
-	DrawInputRecordingOverlay();
+	float position_y;
```

Comment:
Initialize this with zero, then in `DrawInputRecordingOverlay()`, add `margin` to it. That way if something gets added in the middle later, it won't break.